inc/TrkInfo.hh

```diff
@@ -69,8 +69,6 @@ namespace mu2e
     int nnullambig =0;  // number of hits without any ambiguity assigned
     int nmat =0, nmatactive =0; // number materials (straw) assigned and used (active) to this fit
     int nseg =0;     // number of trajectory segments
-    float t0 =0;      // time this particle was estimated to cross z=0
-    float t0err =-1;   // error on t0
```

Comment:
So I think I would technically consider this a breaking change. However, the v04_00_00 tag is so recent that I doubt people have updated their analysis scripts so I propose we don't update the major version number with the next tag.